lambdas/shared/t4_lambda_shared/utils.py

```diff
@@ -15,6 +15,8 @@
 
 PACKAGE_INDEX_SUFFIX = "_packages"
 
+LAMBDA_TMP_SPACE = 512 * 2 ** 20
```

Comment:
AWS says 512 MB not 512 MiB
```suggestion
LAMBDA_TMP_SPACE = 512_000_000
```

Reply:
I tried to run `shutil.disk_usage('/tmp/')` and got this:
```
usage(total=551346176, used=892928, free=538333184)
```
```
In [23]: 538333184 / 2 ** 20
Out[23]: 513.39453125
```

I think it might make sense to drop that check and simply try to preallocate space using [`os.posix_fallocate()`](https://docs.python.org/3/library/os.html#os.posix_fallocate) to fail early.

Reply:
`os.posix_fallocate()` fails with `PermissionError` in lambda :disappointed: .